rules/jsll/src/rules/jsll-awa-init/jsll-awa-init.ts

```diff
@@ -84,12 +85,10 @@ const rule: IRuleBuilder = {
             hasJSLLScript = isJsllDir(element);
         };
 
-        const validateInit = async (event: ITraverseUp) => {
+        const enterBody = async (event: ITraverseUp) => {
```

Comment:
Can you change the type of the ` event`. This is a leftover of when I changed your code.